MAUI/Masked-Entry/Validation.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title: Validation in .NET MAUI MaskedEntry control | Syncfusion
+description: Learn here all about Validation support in Syncfusion .NET MAUI Masked Entry (SfMaskedEntry) control and more.
+platform: maui
+control: SfMaskedEntry
+documentation: ug 
+---
+# Validation in in .NET MAUI Masked Entry (SfMaskedEntry)
+
+## Validation Mode
+
+Input validation happens based on the value of the `ValidationMode`property. The enum values of this property are:
```

Comment:
`ValidationMode`property. -> provide space

Reply:
As we are using the API property here, can I include a space for that?